cypress/support/step-definitions/common-steps.js

```diff
@@ -431,6 +431,10 @@ When('I press ESC onto focused element', () => {
   cy.focused().trigger('keydown', { keyCode: 27, which: 27 });
 });
 
+When('I press Tab onto focused element', () => {
```

Comment:
I don't think that we need a new step, because we have a reusable step in common-steps.js
```
When('I press {string} onto focused element', (arrow) => {
  cy.focused().trigger('keydown', keyCode(arrow));
});
```

Reply:
agree, deleted